media_cdn/snippets.py

```diff
@@ -23,8 +23,12 @@
 
 # [START mediacdn_sign_url]
 # [START mediacdn_sign_cookie]
+# [START mediacdn_sign_token]
 import base64
 import datetime
+import hashlib
+import hmac
```

Comment:
@msampathkumar These two new imports (hashlib and hmac) are only used by 'mediacd_sign_token' sample. I suggest to exclude them from other region tags to make sure samples published to docs include only necessary imports. WDYT?

Reply:
LGTM. I shall update it ASAP.